flaskProject1/static/css/class11.css

```diff
@@ -0,0 +1,118 @@
+body{
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  min-height: 100vh;
+  background-image: url(../images/Classes.jpg );
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.container{
+  flex:1;
+}
+.content{
+  background-color:rgba(255,255,255,255);
+  border-radius:.25em;
+  box-shadow:0 0 .25em rgba(0, 0, 0, 0.25);
+  box-sizing:border-box;
+  left:50%;
+  padding: 5vmin 10vmin;
+  position:fixed;
+  text-align:center;
+  top:50%;
+  transform:translate(-50%, -50%);
+  margin-top: -25px;
+  height: 685px;
+}
+
+.Footer{
+  text-align:center;
+  height: 55px;
+}
+
+i{
+  margin-top: 10px;
+}
+
+h1{
+  margin-top: -10px;
+  margin-bottom: 15px;
+  color: #373b3e;
+}
+
+h2{
+  float: left;
+  margin-left: -60px;
+  color: #373b3e;
+}
+
+#border{
+  float: left;
+  margin-top: 10px;
+  margin-bottom: 10px;
+    /*margin-left: -20px;*/
+  border: solid 3px #373b3e;
+  border-radius: 3px;
+  height: 50px;
+  width: 60vw;
+}
+
+#border a{
+  font-size: 30px;
+  float: left;
+  margin-top: -25px;
+  color: #373b3e;
+  padding-left: 10px;
+  padding-right: 27vw;
+}
```

Comment:
The last name of the subject is not styled correctly. I think the problem is from this css component.